addon/templates/components/html-select.hbs

```diff
@@ -1,4 +1,4 @@
-<select {{action 'change' on='change'}} class="form-control">
+<select {{action 'change' on='change'}} class="{{mainComponent.elementClass}}">
```

Comment:
What I meant was that it may gonna destroy bootstraps standard styling. (http://getbootstrap.com/css/#forms-example)
Also can you edit the dummy app / live examples and add an example with your addition + write docs? Would be nice! I gonna push that later.